rss2email/feed.py

```diff
@@ -478,6 +478,9 @@ def _process_entry(self, parsed, entry):
                 ('Date', self._get_entry_date(entry)),
                 ('Message-ID', '<{}@dev.null.invalid>'.format(_uuid.uuid4())),
                 ('User-Agent', self.user_agent),
+                ('List-ID', '<{}.localhost>'.format(self.name)),
+                ('List-Archive', '<{}>'.format(self.url)),
```

Comment:
I'm just not sure about `List-Archive`, as here it doesn't really point to the archives, but just to the feed. I'm also not sure what benefit adding this header brings?

Reply:
It's not really useful and it's not pointing to a proper archive indeed.
I removed it.